payments-core/src/main/java/com/stripe/android/CardBrandFilter.kt

```diff
@@ -0,0 +1,19 @@
+package com.stripe.android
+
+import android.os.Parcelable
+import androidx.annotation.RestrictTo
+import com.stripe.android.model.CardBrand
+import kotlinx.parcelize.Parcelize
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+interface CardBrandFilter : Parcelable {
+    fun isAccepted(cardBrand: CardBrand): Boolean
```

Comment:
This appears unused (outside of tests).

I'm wondering what your plan is to use this, and if this is the right location/approach for it.

Reply:
Yep good flag it is not used in this PR! I'm pretty confident this is a reasonable approach. The idea is to create a class that given a configuration can tell you if a brand is accepted, and making it easy to test. This type will be used in various places.

Reply:
The thing I'm wondering, is should this be in the payments-core module, or in paymentsheet module. Seems like it shouldn't be needed in payments-core. So that's the part I'm unsure about.

Reply:
Ah I see, I'm pretty certain that it should be in `payments-core`, I believe `GooglePayJsonFactory.kt` and `CardAccountRangeService.kt` (for CBC I think) will need some concept of a type that can tell if a card brand is accepted. If this ends up not being the case I'm happy to relocate.